LithoUtils/Classes/Base/Functions/OpenURL.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  OpenURL.swift
+//  LithoUtils
+//
+//  Created by Calvin Collins on 11/16/21.
+//
+
+import Foundation
+import LithoOperators
+import UIKit
+
+public let openURL = URL.init(string:) >?> (([:], nil) -**> UIApplication.shared.open)
```

Comment:
This is good, but let's make it `openUrl` to differentiate it from the `URL` class and subtly let consumers know it can accept a string